src/main/java/org/jabref/gui/frame/MainMenu.java

```diff
@@ -146,13 +147,19 @@ private void createMenu() {
         Menu tools = new Menu(Localization.lang("Tools"));
         Menu help = new Menu(Localization.lang("Help"));
 
+        MenuItem saveAllMenuItem = factory.createMenuItem(StandardActions.SAVE_ALL, new SaveAllAction(frame::getLibraryTabs, preferences, dialogService));
+        MenuItem newEntryFromPlainTextMenuItem = factory.createMenuItem(StandardActions.NEW_ENTRY_FROM_PLAIN_TEXT, new PlainCitationParserAction(dialogService));
+
+        saveAllMenuItem.disableProperty().bind(Bindings.isEmpty(stateManager.getOpenDatabases()));
+        newEntryFromPlainTextMenuItem.disableProperty().bind(Bindings.isEmpty(stateManager.getOpenDatabases()));
```

Comment:
Reorder:

Variable declaration and binding should be grouped.

Now you have

- A-init
- B-init
- empty line
- A-binding
- B-binding

It shoudl be

- A-init
- A-binding
- empty line
- B-init
- B-binding

Reply:
ok, I will change it.